### PR TITLE
Issue #4985 Adding to troubleshooting section from wiki pages

### PR DESCRIPTION
**Pull request by Loquacity**

### CODE REVIEW

File: adoc/bp_chap_troubleshooting.adoc [hunk 1]
```diff
@@ -402,6 +402,53 @@ If limiting operations to less than a few minutes, you risk aborting perfectly n
...
+
+On SUSE Linux Enterprise Server, use the [command]``zypper locks`` command.
+
+
+
+[[bp.troubleshooting.Corrupt.Repo.Data]]
+== Corrupted Repository Data
+
+
+If the information in [path]``/var/cache/rhn/repodata/sles11-sp2-updates-x86_64`` becomes out of date, it will cause problems with updating the server.
```
Comment (kkaempf) [inline adoc/bp_chap_troubleshooting.adoc]: Pls use a more recent distribution as example, like `sles12-sp3-updates-x86_64`

File: adoc/bp_chap_troubleshooting.adoc [hunk 2]
```diff
@@ -402,6 +402,53 @@ If limiting operations to less than a few minutes, you risk aborting perfectly n
...
+. Regenerate the file with `spacecmd softwarechannel_regenerateyumcache sles11-sp2-updates-x86_64`
+
+
+
+[[bp.troubleshooting.Local.Issuer.Certificate]]
+== Unable to Get Local Issuer Certificate
+
+
+Some older bootstrap scripts will will create a link to the local certificate in the wrong place, which can cause problems with zypper returning an ``Unrecognized error`` about the local issuer certificate.
+In this case, ensure that the link to the local issuer certificate has been created in [path]``/etc/ssl/certs/``, and consider updating your bootstrap scripts.
```
Comment (kkaempf) [inline adoc/bp_chap_troubleshooting.adoc]: @jcayouette don't we have a general advise to update bootstrap scripts regularly ?

Comment (Loquacity) [reply adoc/bp_chap_troubleshooting.adoc]: Even if we do, I think it's OK to mention this as a specific troubleshooting tip. Happy to bow to others with more experience on the product, though!

File: adoc/bp_chap_troubleshooting.adoc (hunk 1 above)
Comment (Loquacity) [reply adoc/bp_chap_troubleshooting.adoc]: https://github.com/SUSE/doc-susemanager/pull/111

**Pull request merged**